src/modules/RF69/RF69.cpp

```diff
@@ -762,6 +787,13 @@ int16_t RF69::setEncoding(uint8_t encoding) {
   }
 }
 
+int16_t RF69::setLnaTestBoost(bool value) {
+  if(value)
```

Comment:
Please use brackets even for single-line branches. It makes it easier to extend them later on.

Reply:
gotcha (and hopefully fixed now?)